src/components/calcite-panel/calcite-panel.tsx

```diff
@@ -177,13 +188,19 @@ export class CalcitePanel {
     );
   }
 
+  renderScrim(): VNode {
+    return this.loading || this.disabled ? (
+      <CalciteScrim loading={this.loading}></CalciteScrim>
+    ) : null;
+  }
+
   render() {
-    const { dismissed, dismissible, el, panelKeyUpHandler } = this;
+    const { disabled, dismissed, dismissible, el, loading, panelKeyUpHandler } = this;
 
     const rtl = getElementDir(el) === "rtl";
 
     return (
-      <Host>
+      <Host aria-busy={loading} aria-disabled={disabled}>
```

Comment:
do booleans work here or does it need text string "true/false"?

Reply:
Seems to be working. I can 2xcheck.

Reply:
Yeah...it works.

Reply:
@kat10140 can you dubz checkz?

Reply:
it twerks @driskull

Reply:
Dah...I need to move those attributes out of Host.
Also, I'll remove the `aria-disabled` attribute like we did in #379.

Reply:
Thanks @kat10140